### PR TITLE
Add kubectl-accurate template list sub command.

### PR DESCRIPTION
**Pull request by d-kuro**
ref: https://github.com/cybozu-go/accurate/issues/3

### CODE REVIEW

File: cmd/kubectl-accurate/sub/template_list.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+func newTemplateListCmd(streams genericclioptions.IOStreams, config *genericclioptions.ConfigFlags) *cobra.Command {
+	opts := &templateListOpts{}
+
+	cmd := &cobra.Command{
+		Use:     "list [TEMPLATE]",
+		Aliases: []string{"ls"},
+		Short:   "List template namespace trees hierarchically",
+		Long: `List template namespace trees hierarchically.
+If TEMPLATE is not given, all root namespaces and their children will be shown.
```
Comment (ymmt2005) [inline cmd/kubectl-accurate/sub/template_list.go:28]:
```suggestion
If TEMPLATE is not given, all template namespaces are shown hierarchically.
```

File: cmd/kubectl-accurate/sub/template_list.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	templates := &corev1.NamespaceList{}
+	if err := o.client.List(ctx, templates, client.MatchingLabels{constants.LabelType: constants.NSTypeTemplate}); err != nil {
+		return fmt.Errorf("failed to list the template namespaces: %w", err)
+	}
+
+	for _, ns := range templates.Items {
+		if err := o.showNS(ctx, ns.Name, 0); err != nil {
+			return err
+		}
+	}
```
Comment (ymmt2005) [inline cmd/kubectl-accurate/sub/template_list.go:70]: Unlike root/sub-namespaces, the above `templates` contain both the root templates and non-root templates.
Try and see some examples.

Here, we should eliminate non-root template namespaces by referencing `accuarte.cybozu.com/template` label.

File: cmd/kubectl-accurate/sub/template_list.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+	subMark := " "
+	if _, ok := ns.Labels[constants.LabelTemplate]; ok {
+		subMark = "⮡"
+	}
+	fmt.Fprintf(o.streams.Out, "%*s%s%s\n", level, "", subMark, name)
+
+	children := &corev1.NamespaceList{}
+	if err := o.client.List(ctx, children, client.MatchingLabels{constants.LabelTemplate: name}); err != nil {
+		return fmt.Errorf("failed to list the children of %s: %w", name, err)
```
Comment (ymmt2005) [inline cmd/kubectl-accurate/sub/template_list.go:88]:
```suggestion
		return fmt.Errorf("failed to list the instances of %s: %w", name, err)
```

**Review by ymmt2005 — CHANGES_REQUESTED** [review]
> Also, add `kubectl-accurate sub list` as an alias for `kubectl-accurate list`.

Please add this as a separate commit.

File: cmd/kubectl-accurate/sub/template_list.go (hunk 2 above)
Comment (d-kuro) [reply cmd/kubectl-accurate/sub/template_list.go]: I see, Thank you follow up.
I fixed:
https://github.com/cybozu-go/accurate/pull/9/commits/10dc7eb114611490206534e18911c6cf96446418

I checked with the following command:

<details>

```console
$ kubectl create ns temp1
$ kubectl create ns temp2
$ kubectl create ns ref1
$ kubectl create ns ref2
$ kubectl create ns ref3

$ go run ./cmd/kubectl-accurate/main.go ns set-type temp1 template
$ go run ./cmd/kubectl-accurate/main.go ns set-type ref1 template
$ go run ./cmd/kubectl-accurate/main.go ns set-type ref2 template
$ go run ./cmd/kubectl-accurate/main.go ns set-type temp2 template
$ go run ./cmd/kubectl-accurate/main.go template set ref1 temp1
$ go run ./cmd/kubectl-accurate/main.go template set ref2 ref1
$ go run ./cmd/kubectl-accurate/main.go template set ref3 temp2

$ go run ./cmd/kubectl-accurate/main.go template list
 temp1
    ⮡ref1
        ⮡ref2
 temp2
    ⮡ref3
```

</details>

File: docs/kubectl-accurate.md [hunk 4]
```diff
@@ -70,6 +70,12 @@ Valid types are `root` or `template`.
 
 To unset the type, specify `none` as the type.
 
+### `template list [TEMPLATE]`
+
+List template namespace trees hierarchically.
+If TEMPLATE is not given, all root namespaces and their children will be shown.
```
Comment (ymmt2005) [inline docs/kubectl-accurate.md:76]:
```suggestion
If TEMPLATE is not given, all template namespaces are shown hierarchically.
```

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: docs/kubectl-accurate.md (hunk 4 above)
Comment (d-kuro) [reply docs/kubectl-accurate.md:76]: Sorry, Thank you comments.

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**